test/runtests.jl

```diff
@@ -13,7 +13,7 @@ for n in [10, 20, 30]
 
       meta = Amd()
       p = amd(A, meta)
-      @test meta.info[AMD_STATUS] == AMD_OK
+      @test meta.info[1] == 0
```

Comment:
Sorry for being annoying. I guess in the tests, you could import those constants, just so it's clear.